src/components/WalletCard.tsx

```diff
@@ -42,7 +42,11 @@ export class WalletCard extends React.Component<Props> {
                 {isAllWallets ? wallet.getLabel() : `${i18n.wallets.dashboard.allWallets}`}
               </EllipsisText>
               {showEditButton && isAllWallets && (
-                <StyledText title={i18n.wallets.details.details} onPress={this.goToWalletDetails} />
+                <StyledText
+                  testID={`show-${wallet.getLabel()}-details-button`}
```

Comment:
I've noticed it doesn't work when the wallet name includes special characters like `'`. Let me know if anyone has a better solution here or simply an idea where to leave a comment informing about the issue

Reply:
I think you should raise a ticket with the issue.

Reply:
I'm afraid it's not an app problem, but Detox one